src/plugins/data/public/ui/query_string_input/query_string_input.tsx

```diff
@@ -584,4 +582,4 @@ export class QueryStringInputUI extends Component<Props, State> {
   }
 }
 
-export const QueryStringInput = injectI18n(withKibana(QueryStringInputUI));
+export const QueryStringInput = withKibana(QueryStringInputUI);
```

Comment:
We can get rid of the variable and rename the class?

Reply:
QueryStringInput and QueryStringInputUI are both used in the other code pieces, so we can't get rid of any of them

Reply:
I think you could replace the existing usages, but its not critical